templates/internal/appName/http/httpservice.go.tpl

```diff
@@ -18,6 +18,14 @@ import (
 	"github.com/getoutreach/gobox/pkg/log"
 	"github.com/getoutreach/httpx/pkg/handlers"
 
+  {{- $additionalImports := stencil.GetModuleHook "internal/http/additionalImports" }}
```

Comment:
Will this get reformatted by `go fmt`, breaking the initial stencil run?

Reply:
We already do this in various other places, like in `rpc.go`, so I think we're good here?